shared/js/media/media_frame.js

```diff
@@ -47,6 +47,8 @@ function MediaFrame(container, includeVideo, maxImageSize) {
   this.image = new Image();
   this.image.className = 'image-view';
   this.image.style.opacity = 0;                        // Start off hidden.
+  this.image.style.transitionProperty = 'opacity';
+  this.image.style.transitionDuration = '100ms';
```

Comment:
Actually, making this animated might mean that calling clear() (which sets opacity back to 0) will trigger an animation. If it works okay without this, it would be safer to just leave it out.  But if you need it, please just test carefully